README.md

```diff
@@ -6,24 +6,24 @@ User applications (e.g. Java/Python/Shell program, Postman) can use the REST API
 
 [Flink JDBC driver](https://github.com/ververica/flink-jdbc-driver) enables JDBC clients to connect to Flink SQL gateway based on the REST API.
 
-Currently, the REST API is internal API and we recommend users to interact with the gateway through JDBC API. 
-Flink SQL gateway stores the session properties in memory now. If the service is stopped or crashed, all properties are lost. We will improve this in future.
+Currently, the REST API is a set of internal API and we recommend users to interact with the gateway through JDBC API. 
+Flink SQL gateway stores the session properties in memory now. If the service is stopped or crashed, all properties are lost. We will improve this in the future.
 
 This project is at an early stage. Feel free to file an issue if you meet any problems or have any suggestions.
 
-# Startup gateway service
+# Startup Gateway Service
 
-There are four steps to starting the service from scratch:
+There are four steps to start the service from scratch:
 
-1. Downloads (or builds) the Flink package. Flink SQL gateway only supports Apache Flink-1.10 now, you can download it through https://flink.apache.org/downloads.html#apache-flink-1100.
+1. Download (or build) the Flink package. Flink SQL gateway currently only supports Apache Flink 1.10, you can download Flink 1.10 from [here](https://flink.apache.org/downloads.html#apache-flink-1100).
 
-2. Starts up a Flink cluster. Flink SQL gateway requires a running Flink cluster where table programs can be executed. For more information about setting up a Flink cluster see the [Cluster & Deployment](https://ci.apache.org/projects/flink/flink-docs-release-1.10/ops/deployment/cluster_setup.html) part.
+2. Start up a Flink cluster. Flink SQL gateway requires a running Flink cluster where table programs can be executed. For more information about setting up a Flink cluster see the [Cluster & Deployment](https://ci.apache.org/projects/flink/flink-docs-release-1.10/ops/deployment/cluster_setup.html) part.
 
-3. Configures the FLINK_HOME environment variable using the following command (and add the same into ~/.bashrc): export FLINK_HOME=&lt;flink-install-dir&gt;
+3. Configure the `FLINK_HOME` environment variable: `export FLINK_HOME=<flink-install-dir>`
```

Comment:
I think `using the following command (and add the same into ~/.bashrc)` is necessary